test/napi/src/js/objects.rs

```diff
@@ -113,12 +113,12 @@ pub fn call_methods_with_prop(mut cx: FunctionContext) -> JsResult<JsString> {
     obj.prop(&mut cx, "setName")
         .bind()?
         .arg("Wonder Woman")?
-        .call()?;
+        .exec()?;
     obj.prop(&mut cx, "toString").bind()?.call()
 }
 
 pub fn call_non_method_with_prop(mut cx: FunctionContext) -> JsResult<JsUndefined> {
     let obj: Handle<JsObject> = cx.argument::<JsObject>(0)?;
-    obj.prop(&mut cx, "number").bind()?.call()?;
+    obj.prop(&mut cx, "number").bind()?.exec()?;
```

Comment:
These are some fly-by lint fixes. `.call()` requires a return generic. Since it's never used, there's an ambiguity where the compiler treats `()` and `!` as equivalent. This will be a compiler error in 2025 edition.